code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -257,4 +257,20 @@
 	if(cell)
 		cell.emp_act(severity)
 
+/obj/item/clothing/head/helmet/space/suicide_act(mob/living/carbon/user)
+	var/datum/gas_mixture/environment = user.loc.return_air()
+	if(HAS_TRAIT(user, TRAIT_RESISTCOLD) || !environment || environment.return_temperature() >= user.get_body_temp_cold_damage_limit())
+		user.visible_message(span_suicide("[user] is beating [user.p_them()]self with \the [src]! It looks like [user.p_theyre()] trying to commit suicide!"))
+		return BRUTELOSS
+	user.say("You want proof? I'll give you proof! Here's proof of what'll happen to you if you stay here with your stuff!", forced = "space helmet suicide")
+	user.visible_message(span_suicide("[user] is removing [user.p_their()] helmet to make a point! Yo, holy shit, [user.p_they()] dead!")) //the use of p_they() instead of p_their() here is intentional
+	user.adjust_bodytemperature(-300)
+	user.apply_status_effect(/datum/status_effect/freon)
+	if(!ishuman(user))
+		return FIRELOSS
+	var/mob/living/carbon/human/humanafterall = user
+	var/datum/disease/advance/cold/pun = new //in the show, arnold survives his stunt, but catches a cold because of it
+	humanafterall.ForceContractDisease(pun, FALSE, TRUE) //this'll show up on health analyzers and the like
```

Comment:
no, not only do they not survive this but i also dont want to have to deal with shitters using eva helmets to commit suicide because something broke and dead bodies can transmit diseases

Reply:
dead bodies can transmit diseases, but that shouldn't be relevant here. the cold virus only contains the sneezing symptom, which can't spread a virus from a corpse without the aid of the necrotic metabolism symptom, since it's spread via the activation of a symptom (instead of being a standard infection vector). furthermore, the cold virus's transmission stat is only 4, which means that it only has access to the blood and fluid contact transmission vectors; you won't catch this virus by just touching the suicider's corpse.

a cold sample is literally available in virology at roundstart; any shitter could just go there to get a sample (or buy a virology crate from cargo, which also contains a cold sample) without needing to commit suicide.

finally, even if someone does catch this virus... it's just sneezing (without the stage 17 threshold met), mate.